plugins/modules/zos_backup_restore.py

```diff
@@ -482,10 +497,11 @@ def backup(
         space_type (str): The unit of measurement to use when defining data set space.
         sms_storage_class (str): Specifies the storage class to use.
         sms_management_class (str): Specifies the management class to use.
+        tmp_hlq (str): Specifies the tmp hlq to temporary datasets
     """
     args = locals()
     zoau_args = to_dzip_args(**args)
-    datasets.zip(**zoau_args)
+    datasets.dzip(**zoau_args)
```

Comment:
I'm curious why the `dunzip` operation in `restore()` below is wrapped in a try/except but this one isn't.

Reply:
Is to keep behavior because zip and zip if fails throw error. But below dzip check if the rc within 1 and 4.